src/tm2/tm-device.h

```diff
@@ -36,6 +36,9 @@ namespace librealsense
         };
         bool compress_while_record() const override { return false; }
 
+    protected:
```

Comment:
I can see the new function's definition but I actually cannot see where and it is applied in the code.
@cchen6 , can you elaborate?

Reply:
@ev-mp it is used on tm2-device.cpp line 1518. tm2-device as sub device to use it for reference to pose sensor, differ from  ds5 device which use first sensor (infrared 1) as reference.